test/e2e/validation_test.go

```diff
@@ -35,6 +35,12 @@ import (
 	"github.com/openshift/windows-machine-config-operator/pkg/windows"
 )
 
+// winService contains information regarding a Windows service's current state
+type winService struct {
```

Comment:
What about exposing and extending the existing [service](https://github.com/openshift/windows-machine-config-operator/blob/22ad459356cb6514095986f6a776870f0a29d57a/pkg/windows/service.go#L6) struct, instead of defining a new one?

Reply:
Or I could be wrong given you have to apply it in the context of the VM object.

Reply:
I revisited this. `svc.setDescription()` is not doable given it needs a reference to the VM object. Passing in the VM object would actually be introducing the inverted dependency. Sorry for the noise.